frontend/src/console/components/RedeploymentToolbar/RedeploymentToolbar.tsx

```diff
@@ -70,73 +76,90 @@ export const RedeploymentToolbar: React.FC<Props> = ({
-            <p>
-              <WrappedMessage
-                id="cancelDeploymentConfirm"
-                messages={messages}
-              />
-            </p>
-          </Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          <WrappedMessage
-            id="cancelDeploymentConfirmText"
-            messages={messages}
-          />
-        </Modal.Body>
-        <Modal.Footer>
-          <Button
-            variant="outline-primary"
-            size="lg"
-            onClick={handleCloseModal}
-          >
-            <WrappedMessage id="closeModalButton" messages={messages} />
-          </Button>
           <Button
+            className="float-right loading"
             variant="primary"
             size="lg"
             onClick={() => {
-              cancelRedeployment();
-              handleCloseModal();
+              performDeployment();
             }}
+            disabled={deploymentDisabled}
           >
-            <WrappedMessage id="confirmCancelDeployment" messages={messages} />
+            <WrappedMessage
+              id="deploy"
+              messages={messages}
+              values={{ undeployedChanges }}
+            />
           </Button>
-        </Modal.Footer>
-      </Modal>
+        </div>
+
+        <Modal
+          show={show}
+          onHide={handleCloseModal}
+          className="cancel-redeployment-modal"
+          centered
+        >
+          <Modal.Header>
+            <Modal.Title>
+              <p>
+                <WrappedMessage
+                  id="cancelDeploymentConfirm"
+                  messages={messages}
+                />
+              </p>
+            </Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <WrappedMessage
+              id="cancelDeploymentConfirmText"
+              messages={messages}
+            />
+          </Modal.Body>
+          <Modal.Footer>
+            <Button
+              variant="outline-primary"
+              size="lg"
+              onClick={handleCloseModal}
+            >
+              <WrappedMessage id="closeModalButton" messages={messages} />
+            </Button>
+            <Button
+              variant="primary"
+              size="lg"
+              onClick={() => {
+                cancelRedeployment();
+                handleCloseModal();
+              }}
+            >
+              <WrappedMessage
+                id="confirmCancelDeployment"
+                messages={messages}
+              />
+            </Button>
+          </Modal.Footer>
+        </Modal>
+      </div>
+      {!isEmailVerified && alertShow ? (
+        <div className="redeployment-toolbar">
+          <Alert
+            className="alert-info"
+            onClose={() => handleCloseAlert()}
+            variant="info"
+            dismissible
+          >
+            <Alert.Heading>Please verify all your emails!</Alert.Heading>
+          </Alert>
+        </div>
+      ) : null}
```

Comment:
I don't think the redeployment toolbar is the right place for this. I think it's better to move this to its own component and have it directly on the ConsolePage.

Reply:
@xitij2000 We need to hide the deployment toolbar before the user activates the email, otherwise it'll wrongfully show that there are pending changes (since there's no changes).

So I though of turning the deployment toolbar into a yellow container with the message stating that the user needs to active the emails before the instance is set up.

Reply:
OK. In that case this can just replace the deployment toolbar?

Reply:
I have made the same :+1: